### ISSUE TITLE
Calls to validation service failing with "Cannot read property 'messages' of undefined"

### ISSUE THREAD

**Issue by rgladwell** [mozilla/html5-lint#25]
Calls to validation service failing with:

> Fatal error: Cannot read property 'messages' of undefined Cannot read property 'messages' of undefined

**Comment by git-no** [comment]
if you add **throw err** line to your code like:

``` js
...
  }, function (err, css) {
    if (err) throw err;
    var minified = new cleanCSS().minify(css).styles;
...
```

execution  should report: **Error: Internal Server Error**
This happend to me too. No clue. Seems server [https://validator.mozillalabs.com/](https://validator.mozillalabs.com/) is down.

**Comment by rgladwell** [comment]
@git-no As far as I can see `html-lint` uses https://html5.validator.nu/.

**Comment by git-no** [comment]
I got that from the [README.md](https://github.com/mozilla/html5-lint) of this repo.
It says: _"This is a node.js and Python front-end to the Mozilla Labs' HTML Validator Web Service, located at https://validator.mozillalabs.com/."_
Is the README wrong?

**Comment by rgladwell** [comment]
Looks as though it is.

**Comment by cbuchert** [comment]
I'm having the same issue (reported on this package's Gulp implementation [here](https://github.com/LiveSafe/gulp-html5-lint/issues/11).

**Comment by rgladwell** [comment]
I suspect the API for html5.validator.nu has changed and broken this client.

**Comment by cbuchert** [comment]
Any updates?

**Comment by boneyfantaseas** [comment]
I installed gulp-html-lint today and had similar problems.
Basically, I get a Bad Request (400) which tells me that there must be a UA-Header attached. 
This is also shown in the example on https://github.com/validator/validator/wiki/Service-%C2%BB-Input-%C2%BB-POST-body.

After fixing this locally the service worked again.
How did you guys work around the issue? Manual validation? Another tool?
